ansible/roles/ocp-workload-starter-workshop/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+- In main.yml needs become:true as the user that has oc configured is root.
```

Comment:
On shared cluster we set up an admin user `opentlc-mgr` that has system:admin user configured for `oc` commands.